client/tests/acceptance/hearings-list-for-milestones-list-shows-up-correctly-test.js

```diff
@@ -425,11 +415,9 @@ module('Acceptance | hearings list for milestones list shows up correctly', func
           milestonename: 'Land Use Application Filed',
           dcpMilestone: '663beec4-dad0-e711-8116-1458d04e2fb8',
           outcome: null,
-          displayDate2: null,
-          displayDate: '2019-07-13T19:31:57.763Z',
           statuscode: 'Completed',
-          dcpActualenddate: null,
-          dcpActualstartdate: '2019-07-13T19:31:57.763Z',
+          dcpActualenddate: '2019-07-13T19:31:57.763Z',
+          dcpActualstartdate: null,
```

Comment:
`Moving the displayDate calculation code to the frontend actually revealed this mistake in the factory setup`, since mirage no longer hardcodes displayDate(2) and hidden from the CP implementation.